pkg/microservice/aslan/core/system/service/install.go

```diff
@@ -157,7 +157,7 @@ func InitInstallMap() map[string]*commonmodels.Install {
 		ObjectIDHex: "5cc4294af92295138e2d8478",
 		Name:        "yarn",
 		Version:     "1.15.2",
-		Scripts:     "sudo apt-get update\napt-get install -y npm\nnpm install -g yarn\nyarn set version 1.15.2",
+		Scripts:     "npm install -g yarn\nyarn set version 1.15.2",
```

Comment:
Inorder to make it work, user need to install node first, it that ok? @AllenShen

Reply:
If there is no simple way to use official bin file, and if this is official way to install yarn, it's OK to me.